### ISSUE TITLE
Installing Dropbox on a headless Ubuntu server

### ISSUE THREAD

**Issue by scripting** [scripting/Scripting-News#169]
I'm still trying to dig out of the mess I have with Dropbox. 

Two of my servers on Digital Ocean, the oldest ones I have, no longer are able to run Dropbox. Everything I've tried to do to get it working has failed. I'm ready to give up on that approach, and move the contents of those servers to a new server, freshly installed with the following:

1. Node and NPM.

2. Forever.

3. Dropbox.

From there I can finish the job. But get this -- I can't get Dropbox to install on a fresh system. I get a Python error. 

I have no idea who's to blame. But I can't recreate the old servers without Forever. 

I follow the instructions on [this page](http://scripting.com/2018/12/19/160039.html). Obviously those instructions used to work. 

The server I'm installing on is 4 GB Memory / 60 GB Disk / NYC1 - Ubuntu 14.04.4 x64.

Here's a screen shot of the error I get.

![image](https://user-images.githubusercontent.com/1686843/80499774-63489b80-893b-11ea-80ca-0ad6646a5de2.png)

**Comment by tedchoward** [comment]
Dave, it looks like the error is about a missing library `libglapi.so.0`. I did a search for that term, and came across this forum post that may be helpful:

https://www.dropboxforum.com/t5/Dropbox-installs-integrations/Headless-install-on-Debian-libglapi-so-0-cannot-open-shared/td-p/396457

**Comment by scripting** [comment]
Ted, thanks -- you were right, and it's funny and sad in a way that the thing I had to fix on all the original servers is what I needed to fix on the fresh one. 

https://www.dropboxforum.com/t5/Dropbox-installs-integrations/Stuck-on-quot-upgrading-dropbox-quot/m-p/414171/highlight/false#M92358

I guess it didn't occur to me that they would *ship* a product with this flaw in it. ;-)

I now will produce a new checklist for provisioning a new server. Stuff changed with Node too, of course. ;-)

Thanks again.
